README.md

```diff
@@ -1 +1 @@
-# my-trial-review
\ No newline at end of file
+# this is my-trial-review
```

Comment:
Hi 😃
good work but there are little required changes that should be completed:

1- Add a descriptive README file to your project - please use [this template ](https://github.com/microverseinc/readme-template) but remember to customize it to your project.